templates/parachain/README.docify.md

```diff
@@ -240,6 +241,127 @@ npx @acala-network/chopsticks@latest --chain-spec <path/to/chain_spec.json>
+
+Upon success, your parachain will start producing blocks.
+
+### Generate Customs Keys for Collator Node
+
+1. To perform this step, you can use subkey, a command-line tool for generating and managing keys:
+```
+docker run -it parity/subkey:latest generate --scheme sr25519
+```
+
+### Build the Chain Specification
+Generate the plain chain specification:
+```
+polkadot-omni-node chain-spec-builder \
+  --chain-spec-path ./plain_chain_spec.json \
+  create \
+  --relay-chain paseo \
+  --para-id <YOUR_PARA_ID> \
+  --runtime target/release/wbuild/parachain-template-runtime/parachain_template_runtime.compact.compressed.wasm \
+  named-preset local_testnet 
+```
+Edit plain_chain_spec.json to update fields like name, id, protocolId, para_id, and parachainInfo.parachainId with your ParaID. Also, configure balances, collatorSelection.invulnerables, session.keys, and sudo fields accordingly.
+Then, generate the raw chain specification:
+
+``` 
+polkadot-omni-node chain-spec-builder \
+  --chain-spec-path ./raw_chain_spec.json \
+  build \
+  --chain plain_chain_spec.json
+```
+
+### Start the Collator Node
+
+1. Before starting a collator, you need to generate a node key. This key is responsible for communicating with peers as in a p2p network:
+
+```
+polkadot-omni-node key generate-node-key \
+--base-path data \
+--chain raw_chain_spec.json
+
+```
+2. You can start the collator with a command similar to the following:
+
+``` 
+polkadot-omni-node --collator \
+--chain raw_chain_spec.json \
+--base-path data \
+--port 40333 \
+--rpc-port 8845 \
+--force-authoring \
+--node-key-file ./data/chains/custom/network/secret_ed25519 \
+-- \
+--sync warp \
+--chain paseo \
+--port 50343 \
+--rpc-port 9988
+```
+
+Ensure that the paths and ports are correctly set according to your environment.
+
+
+### Obtain Coretime
+
+To produce blocks, your parachain needs coretime. You can acquire it in two ways:
+
+1. On-Demand Coretime: Use the onDemand.placeOrderAllowDeath extrinsic on the Paseo relay chain. In Polkadot.js Apps, select the extrinsic, input your ParaID and desired amount, and submit the transaction.
+2. Bulk Coretime: Purchase via the Broker pallet on the coretime system parachain. Assign the purchased core to your registered ParaID.
+
+Once coretime is assigned, your collator should start producing blocks. Monitor the logs to confirm block production.
+
+For a more streamlined deployment experience, consider using the [Polkadot Deployment Portal (PDP)](https://polkadot.polkassembly.io/forum/t/polkadot-deployment-portal-the-1-click-solution-for-polkadot/12176), which simplifies the process of deploying parachains and managing coretime.
+
+For more detailed guidance, refer to the [Zero to Hero: Deploy on Paseo TestNet](https://docs.polkadot.com/tutorials/polkadot-sdk/parachains/zero-to-hero/deploy-to-testnet/) and [Obtain Coretime](https://docs.polkadot.com/tutorials/polkadot-sdk/parachains/zero-to-hero/obtain-coretime/) tutorials.
```

Comment:
Isn't this tackling same subject as `Acquire Coretime` section? Might need to be merged?